lib/index.js

```diff
@@ -1,13 +1,13 @@
 "use strict";
 
 const emojer = require("emojer")
-    , emojis = require("github-api-emojis/lib/map")
+    , emojis = require("github-api-emojis/lib/map.json")
 
-module.exports = function showdownEmoji () {
-    return [
-        {
-            type: "output"
-          , filter: text => emojer(text, "<img src=\"https://github.global.ssl.fastly.net/images/icons/emoji/__EMOJI_NAME__.png?v5\" alt=\":__EMOJI_NAME__:\" title=\":__EMOJI_NAME__:\" class=\"emoji-img emoji\">", emojis)
+module.exports = function showdownEmoji() {
+    return [{
+        type: "output",
+        filter: function filter(text) {
```

Comment:
Umm, is there a specific reason for using ES5?

Reply:
I'm sorry. Webpack3 can't process uglifyJS with ES6. And I don't know how to transpile that code with babel.